feeadjuster/test_feeadjuster.py

```diff
@@ -216,3 +216,76 @@ def test_feeadjuster_imbalance(node_factory):
         f'Set default fees as imbalance is too low: {scid_B}'
     ])
     wait_for_fees(l2, scids, default_fees[0])
+
+
+@unittest.skipIf(not DEVELOPER, "Too slow without fast gossip")
+def test_feeadjuster_big_enough_liquidity(node_factory):
+    """
+    A rather simple network:
+
+            A                   B
+    l1  <========>   l2   <=========>  l3
+
+    l2 will adjust its configuration-set base and proportional fees for
+    channels A and B as l1 and l3 exchange payments.
+    """
+    base_fee = 5000
+    ppm_fee = 300
+    l2_opts = {
+        "fee-base": base_fee,
+        "fee-per-satoshi": ppm_fee,
+        "plugin": plugin_path,
+        "feeadjuster-deactivate-fuzz": None,
```

Comment:
Hm, will this test work better with `"feeadjuster-deactivate-fuzz": True` ?

Reply:
In theory, it shouldn't matter. The fuzz is very little in this case, and the forwarded amount is far bigger.

In practice, I tried with `"feeadjuster-deactivate-fuzz": True`, but caused errors:
```
lightningd: --feeadjuster-deactivate-fuzz: doesn't allow an argument
```
Maybe it isn't how it works. Maybe `"feeadjuster-deactivate-fuzz": None` means the switch is turned on. (Kind of counterintuitive.) So I tried out what happens if I remove this line. But on my machine it does not matter, the tests are pass with or without this line.

I don't know why CI fails.

Reply:
I try to rebase to origin/master to see if it helps.

Reply:
It is not the rebase, but I've figured out a way to make CI work.
However, it's a bit frustrating that I cannot reliably test it on my machine because the CI test can differ.

Reply:
I think your right on `"feeadjuster-deactivate-fuzz": None` means turning it on :D